### PR TITLE
[496] feat: added design for help desk page

### PR DESCRIPTION
**Pull request by Sperthix**
**Changes proposed in this Pull Request**
Changed design of help desk landing page as in [figma file](https://www.figma.com/file/MyPe8HfTYEoOzmxMpdJQ1h/LA-WEB-RfD---Helpdesk?node-id=139%3A128). Added design for solutions element separatly.

**Testing instructions**
Please chceck design in various browsers on [this](http://liveagent.local/help-desk-software-redesign/) page

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#496

### CODE REVIEW

File: assets/styles/components/_BlockSolution.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/* stylelint-disable selector-class-pattern */
+
+.BlockSolution {
+
+	.elementor-inner-section {
+		position: relative;
+	}
+
+	.business-solution-wrapper {
+		max-width: 1400px !important;
```
Comment (jremen) [inline assets/styles/components/_BlockSolution.scss:10]: if possible, use $breakpoint-desktop-box instead. We have max width 1380px united across all web

File: assets/styles/components/_BlockSolution.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+	.column--solution {
+		position: relative;
+		max-width: 300px !important;
+		margin-left: 1em;
+
+		&::after {
+			content: "";
+			position: absolute;
+			background: rgb(255, 255, 255);
```
Comment (jremen) [inline assets/styles/components/_BlockSolution.scss:21]: background: $white

File: assets/styles/components/_BlockSolution.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+			border-radius: 16px;
+			width: 100%;
```
Comment (jremen) [inline assets/styles/components/_BlockSolution.scss:23]: @include square(100%);
border-radius: $border-radius-16

File: assets/styles/components/_BlockSolution.scss [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+			height: 100%;
+			z-index: 1;
+		}
+
+		&-left {
+			position: relative;
+			width: 29% !important;
```
Comment (jremen) [inline assets/styles/components/_BlockSolution.scss:30]: avoid important if possible (most of time, it is)

File: assets/styles/layouts/_HelpDesk.scss [hunk 5]
```diff
@@ -0,0 +1,91 @@
+/* stylelint-disable selector-class-pattern, no-descending-specificity */
+
+.HelpDesk {
+
+	&--wide {
+		max-width: 1380px !important;
```
Comment (jremen) [inline assets/styles/layouts/_HelpDesk.scss:6]: breakpoint-desktop-box, remove important

**Review by jremen — CHANGES_REQUESTED** [review]

**Review by jremen — APPROVED** [review]

**Review by jremen — APPROVED** [review]

**Review by mvstepanek — APPROVED** [review]

**Pull request merged**